lib/rubocop/cop/lint/multiple_comparison.rb

```diff
@@ -34,6 +35,7 @@ class MultipleComparison < Base
 
         def on_send(node)
           return unless (center = multiple_compare?(node))
+          return if center.send_type? && SET_OPERATION_OPERATORS.include?(center.method_name)
```

Comment:
I think we'll definitely have to explain here, why those set operations matter at all.